app/src/main/res/drawable-night/ic_empty_collectibles.xml

```diff
@@ -0,0 +1,25 @@
+    <path
+        android:pathData="M117.877 11C118.481 11 119.049 11.2724 119.427 11.7358L119.525 11.8668L134.012 32.9348C134.276 33.3187 134.384 33.7427 134.366 34.1515L134.347 34.3488L134.343 34.3719C134.324 34.4926 134.294 34.6112 134.254 34.7262L134.175 34.9175L101.98 103.54L101.917 103.659C101.578 104.256 101.001 104.586 100.4 104.65C100.317 104.659 100.232 104.662 100.147 104.661L99.9786 104.651L99.8494 104.633C99.2934 104.542 98.7717 104.22 98.4542 103.667L98.3728 103.51L66.1822 34.8946C65.93 34.341 65.9239 33.6734 66.2634 33.0811L66.3555 32.9347L80.8435 11.8667C81.1855 11.3694 81.7319 11.0555 82.3278 11.0067L82.4914 11H117.877ZM87.194 36.067L100.183 93.582L113.172 36.067H87.194ZM71.1503 36.0671L83.094 36.067L94.181 85.158L71.1503 36.0671ZM129.216 36.0671L117.273 36.067L106.187 85.153L129.216 36.0671ZM83.5433 14.9991L90.685 14.999L83.374 32.067L71.8063 32.0671L83.5433 14.9991ZM105.331 14.999H95.037L87.725 32.067H112.641L105.331 14.999ZM109.682 14.999L116.824 14.9991L128.561 32.0671L116.992 32.067L109.682 14.999Z"
+        android:fillType="evenOdd"
+        android:fillColor="#636669" />
+    <path
+        android:pathData="M66.0011 22C64.9622 22 64.0509 22.6835 63.7586 23.6774L63.5861 24.2682C62.083 29.3267 61.0585 30.2123 55.6914 31.7551C54.6934 32.0392 54 32.9553 54 34C54 35.0414 54.6896 35.9524 55.6833 36.2426L56.2734 36.4148C61.3263 37.9159 62.2122 38.9411 63.7551 44.3096C64.0491 45.3166 64.9639 45.9989 66.0011 45.9989C67.04 45.9989 67.9598 45.3112 68.2463 44.3048L68.5026 43.4347C69.9575 38.6389 71.0434 37.7575 76.3093 36.245C77.3141 35.9537 78 35.0392 78 34C78 32.9575 77.3103 32.0379 76.3011 31.7527L75.4318 31.4965C70.6401 30.0429 69.7587 28.9566 68.2449 23.6893C67.958 22.6889 67.0416 22 66.0011 22Z"
+        android:fillColor="#12FF80" />
+    <path
+        android:pathData="M38 28C38 22.4772 33.5228 18 28 18C22.4772 18 18 22.4772 18 28C18 33.5228 22.4772 38 28 38C33.5228 38 38 33.5228 38 28ZM22 28C22 24.6863 24.6863 22 28 22C31.3137 22 34 24.6863 34 28C34 31.3137 31.3137 34 28 34C24.6863 34 22 31.3137 22 28Z"
+        android:fillType="evenOdd"
+        android:fillColor="#636669" />
+    <path
+        android:pathData="M60.2448 73.3369C60.2368 73.6615 60.1482 73.9909 59.965 74.2972L59.8708 74.4413L37.2112 104.165C36.4441 105.171 34.9609 105.213 34.1341 104.291L34.0301 104.165L11.442 74.5354C11.4053 74.4894 11.3706 74.4417 11.338 74.3927C11.0806 74.0106 10.9773 73.5875 11.0001 73.1793C11.0136 72.8806 11.0965 72.5791 11.2589 72.2957C11.3098 72.2071 11.3661 72.1225 11.4286 72.0425L20.6601 59.5697C21.003 59.1064 21.5271 58.8159 22.0958 58.7669L22.2677 58.7595H49.8887C50.5014 58.7595 51.0771 59.0402 51.4545 59.5153L51.5524 59.6496L59.9587 72.2547C60.1443 72.5634 60.2347 72.8931 60.2442 73.2172L60.2448 73.3369ZM54.2 75.2805H17.04L35.621 99.6539L54.2 75.2805ZM23.276 62.7589H48.818L54.502 71.2799H16.969L23.276 62.7589Z"
+        android:fillType="evenOdd"
+        android:fillColor="#12FF80" />
+</vector>
```

Comment:
![image](https://user-images.githubusercontent.com/17750984/192994337-35a83c7e-7f6e-4359-9773-8d8638bfc18f.png)
Make sure to enable `Ensure every saved file ends with a line break` option

Reply:
I have it switched on. But these files were produced by a tool that converted all svg files in a folder at once. So I probably have to go through all files and safe them once.